README.md

```diff
@@ -76,7 +76,7 @@ The following instructions describe how to set up the pet clinic sample applicat
 
    - Install `java17`, `git`, and `tmux` as dependencies
```

Comment:
Update comment:
```suggestion
   - Install `java17` and `tmux` as dependencies
```

Reply:
Actually we want to install git in this case. We will need to use git to clone the repo.

Reply:
Yep, misread, sorry.